src/include/daos/drpc_modules.h

```diff
@@ -86,6 +86,7 @@ enum drpc_mgmt_method {
 	DRPC_METHOD_MGMT_DEV_IDENTIFY		= 234,
 	DRPC_METHOD_MGMT_NOTIFY_POOL_CONNECT	= 235,
 	DRPC_METHOD_MGMT_NOTIFY_POOL_DISCONNECT	= 236,
+	DRPC_METHOD_MGMT_CLUSTER_EVENT		= 237,
```

Comment:
As noted earlier, I don't think this is a `_MGMT_` method, it's a `_SRV_` method.

Reply:
will change